test/kind-config.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2020, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    image: kindest/node:v1.16.9@sha256:7175872357bc85847ec4b1aba46ed1d12fa054c83ac7a8a11f5c268957fd5765
```

Comment:
OKE is now creating 1.17.9 clusters by default.  Maybe we should update these node images to be 1.17.

kindest/node:v1.17.11@sha256:5240a7a2c34bf241afb54ac05669f8a46661912eab05705d660971eeb12f6555

Reply:
I would prefer to do that in subsequent JIRA since this is working.

Reply:
fixed